act-rules-format.bs

```diff
@@ -109,38 +109,34 @@ An aspect is a distinct part of the [test subject](#output-test-subject) or its
 
 An atomic rule MUST include a description of all the aspects under test by the rule. Each aspect MUST be discrete with no overlap between the aspects. Some aspects are already well defined within the context of web content, such as HTTP messages, DOM tree, and CSS styling [[CSS2]], and do not warrant a detailed description. Other aspects may not be well defined or even specific to web content. In these cases, an ACT Rule SHOULD include either a detailed description of the aspect in question or a reference to that description.
 
+A list of common aspects for reference in ACT rules can be found in [Common Aspects under Test](#).
```

Comment:
@nitedog can you create a page for us to link to somewhere on w3.org?

Reply:
how do you feel about a WG Note instead?

Reply:
I think that works. @maryjom any thoughts?

Reply:
@nitedog Can you explain what the difference would be between a WG Note, an Understanding doc, and simply putting it in a non-normative Appendix.  Just trying to minimize the number of documents we need to maintain and understand the right place to develop/put such content.

Reply:
A WG Note is non-normative. This is fairly easy to write, publish, and update,, and is more formal than "just any page on the W3C website". At the same time, it is not part of the primary ACT Rules Format specification, which is a normative W3C Recommendation and significantly harder to update (eg. when we identify new aspects).

Actually, I foresee three or four WG Notes supporting the primary ACT Rules Format specification:
- Collection of "Aspects" (eg. HTTP etc. taken from the current spec)
- Generic review process (which we already have https://w3c.github.io/wcag-act-rules/review-process.html)
- Suggested notation for output format (which we also have at https://github.com/w3c/earl)
- (possibly) suggested template for rules (also available https://auto-wcag.github.io/auto-wcag/pages/design/rule-template.html)

All these WG Notes would be referenced in a non-normative way (eg. "you MAY use the following" type of requirement). We already have most of the content but it is rather scattered. This would pull all the content into one "package" with coherent cross-referencing and linking.

Could you please add this for discussion at one of the next calls?